public/locales/en/files.json

```
@@ -45,6 +45,7 @@
   },
   "pinningModal": {
     "title": "Select where you would like to pin these items.",
+    "complianceLabel": "Check pinning services' compliance",
```

```suggestion
    "complianceLabel": "🔍 Check pinning services' compliance",
```

Here and elsewhere, the links my subtly pop more with an emoji like 🔍